### PR TITLE
Can set multiple search pattern,separate by comma

### PR DESCRIPTION
**Pull request by shelocks**

### CODE REVIEW

File: cmd/bosun/sched/template.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func (c *Context) eval(v interface{}, filter bool, series bool, autods int) (exp
 		return nil, "", fmt.Errorf("%v: %v", v, err)
 	}
 	if filter {
-		e, err = expr.New(opentsdb.ReplaceTags(e.String(), c.State.Group))
+		e, err = expr.New(opentsdb.ReplaceTags(e.Text, c.State.Group))
```
Comment (mjibson) [inline cmd/bosun/sched/template.go]: This commit is totally unrelated to the rest. Could you please submit it as a separate PR? And remove it from this PR. I'd appreciate squashing all these commits together into a single commit, too.

**Comment by shelocks** [comment]
remove unrelated commit from this pr

**Comment by shelocks** [comment]
remove unrelated commit from this pr

**Comment by mjibson** [comment]
Please squash into one commit and force push.

**Comment by shelocks** [comment]
squash into single commit

**Comment by mjibson** [comment]
@shelocks Thanks. I'd like to fix that other .Text thing, too, but I forget exactly the problem it was causing? What was that?

**Comment by shelocks** [comment]
Submit a new PR#637
